### ISSUE TITLE
Is burn in working as intended?

### ISSUE THREAD

**Issue by rhshi** [k-r-allen/residual-policy-learning#13]
I have a few questions on this as well. What exactly is skip training doing? The default is that it is True, so regardless of the burn in the model seems to not train at all. Also, the critic should be "pre-trained," correct? However, the way it is set up seems to be that it can always set the policy learning rate to 0 - during training, maybe the critic suddenly shows poorer performance, and so it is possible that this if statement is true. Lastly, why take the difference in losses/it seems that the actor loss is what is actually being pre-trained (you have losses.append(actor_loss) in the training loop).  Thanks!

_Originally posted by @rhshi in https://github.com/k-r-allen/residual-policy-learning/issues/12#issuecomment-530667931_
